.github/workflows/ubuntu.yml

```diff
@@ -145,11 +142,11 @@ jobs:
       fail-fast: false
       matrix:
         combo:
```

Comment:
We benefit from `latest`, consistently, and thus this should break much less in the future.